python/GangaLHCb/__init__.py

```diff
@@ -85,43 +85,21 @@ def _store_root_version():
 
 
 def _store_dirac_environment():
+    from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache
     diracversion = _guess_version('LHCBDIRAC')
-    import tempfile
-    import subprocess
-    import os
     platform = os.environ['CMTOPT']
     setup_script = 'SetupProject.sh'
```

Comment:
That's what I've tested and indeed works locally. Do we want a switch to be able to select either based on a config or should we just switch to `lb-run` wholly?